backend/fms_core/filters.py

```diff
@@ -173,6 +174,16 @@ class Meta:
         model = Dataset
         fields = _dataset_filterset_fields
 
+class ReadsetFilter(GenericFilter):
+    has_released_files = django_filters.BooleanFilter(method="has_released_files_filter")
```

Comment:
I wonder if this should be a count of the number of released files, rather than a boolean.

Reply:
I'm a bit skeptical of that because then the user would have to provide a filter checking if count is greater than a certain value, and the count is not part of the readset serializer (yet).

Reply:
So now the question becomes should we include datasetfile count as part of the ReadsetSerializer?

Reply:
Oh, right. I was confusing this filter with the serializer.